goal_src/jak2/engine/ui/progress/progress-static.gc

```diff
@@ -7,3 +7,936 @@
+
+(define *sound-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *quit-restart-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *load-save-options* (new 'static 'menu-option-list :y-center #xdc :y-space 30 :scale 0.82))
+
+(define *save-options-title* (new 'static 'menu-option-list :y-center #xdc :y-space 30 :scale 0.82))
+
+(define *loading-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *insufficient-space-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *secrets-insufficient-space-options*
+  (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82)
+  )
+
+(define *insert-card-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *error-loading-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *error-auto-saving-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *card-removed-options* (new 'static 'menu-option-list :y-center #x104 :y-space 30 :scale 0.82))
+
+(define *error-disc-removed-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *error-reading-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *icon-info-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *format-card-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *already-exists-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *create-game-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *video-mode-warning-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *video-mode-ok-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *progressive-mode-warning-options*
+  (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82)
+  )
+
+(define *progressive-mode-ok-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *quit-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *select-start-options*
+  "List of [[menu-select-start-option]]"
+  (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82)
+  )
+
+(define *select-scene-options*
+  "List of [[menu-select-scene-option]]"
+  (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82)
+  )
+
+(define *bigmap-options* (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82))
+
+(define *missions-options*
+  "List of [[menu-missions-option]]"
+  (new 'static 'menu-option-list :y-center #xc6 :y-space 30 :scale 0.82)
+  )
+
+(define *highscores-options*
```

Comment:
Are these actually correct? I'd guess that the `options` array isn't empty on all of these. This might need some special case in data_decompile.cpp

Reply:
hmmm yeah...something seems wrong now, also the file now suddenly has label issues.  I'll look into this -- the one you are referencing here for example should have an `array` of 1 element

Reply:
Yeah looks like this is an unimplemented case https://github.com/open-goal/jak-project/blob/9a04c7e3116283271a371d121dcdae199baab10b/decompiler/util/data_decompile.cpp#L1192 im not quite sure how complex this would be to implement?  In this case i imagine it's just a copy-paste of our code that already handles normal arrays -- but i imagine it's likely `:dynamic` fields aren't always so nice?